ietfYangDraftPull/draftPull.py

```diff
@@ -41,35 +40,22 @@
 from ietfYangDraftPull import draftPullUtility as dpu
 from utility import message_factory
 from utility.create_config import create_config
-from utility.scriptConfig import Arg, BaseScriptConfig
+from utility.script_config_dict import script_config_dict
+from utility.scriptConfig import ScriptConfig
 from utility.util import job_log
 
+BASENAME = os.path.basename(__file__)
```

Comment:
In some files there is already line of code `current_file_basename = os.path.basename(__file__)` which is same as `BASENAME = os.path.basename(__file__)`
Please use only one of these variables - I would prefer `BASENAME`